Server/src/controllers/DataSetController.ts

```diff
@@ -0,0 +1,245 @@
+import { Request, Response } from 'express';
+import { DataSetService } from '../services/DataSetService';
+import { IDataRequestModel } from "../models/interfaces/DataRequestModelInterface";
+
+export class DataSetController {
+    private processedRequest: IDataRequestModel;
+    private dataSetService: DataSetService
+
+    constructor() {

```

Comment:
is it a requirement to add an empty constructor?

Reply:
Yeah. Or else we must have static methods